src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useContext } from "react";
+import {
+	ACTIONS,
+	EpisodesContext as Episodes,
+} from "COMPONENTS/Episodes/";
+import { TrailerContext as Trailer } from "COMPONENTS/Episode/";
+import UTILS from "SHARED/utils.js";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	// ----------------------------
+	const {
+		state: {
+			activeIndex,
+			overlayActive,
+			episodeCount
+		},
+		dispatch
+	} = useContext(Episodes);
+
+
+	const trailers = useContext(Trailer);
+
+	const { src: prevTrailerSrc } = trailers.prev || {};
+	const { src: nextTrailerSrc } = trailers.next || {};
```

Comment:
Let's do:

```
const { 
    prev: prevTrailer = {}, 
    next: nextTrailer = {}
} = useContext(Trailer);

const { src: prevTrailerSrc } = prevTrailer;
const { src: nextTrailerSrc } = nextTrailer;
```

That way we can use the whole 'default' value stuff that comes with destructuring instead of the `|| {}` fallback, which is less explicity 'default'.